repose-aggregator/components/filters/destination-router-filter/src/main/scala/org/openrepose/filters/destinationrouter/DestinationRouterFilter.scala

```diff
@@ -95,7 +91,13 @@ class DestinationRouterFilter @Inject()(configurationService: ConfigurationServi
             logger.error("The destination could not be added -- the destinations attribute was of an unknown type")
         }
 
-        mbcsRoutedReponse.mark(target.getId)
+        metricsServiceOption.foreach(metricsService =>
```

Comment:
So i haven't looked at what you changed it to yet, but I thought about it more last night, and maybe a method on the service that took the arguments and did all the calls? It seems reasonable since you do the exact same thing on all of these calls. So it'd be `metricsService.mark(whatever.class, bunchOfStringsVarArg)` I think there was another arguement in there, but i don't remember.